utilities/transactions/pessimistic_transaction.cc

```diff
@@ -630,7 +630,12 @@ Status PessimisticTransaction::TryLock(ColumnFamilyHandle* column_family,
     // the key is already locked, this func will update some stats on the
```

Comment:
It would be great to add some inline comments to the stats as their purpose has been unclear for me up until now. It seems that their only significance is to clarify if there has been at least a read/write since last save point so that we could gc tracked key during rollbacktosavepoint if there has not.